internal/mithril-persistence/src/store/mod.rs

```diff
@@ -1,9 +1,6 @@
 //! Define a generic way to store data with the [Store Adapters][adapter], and the [StakeStorer]
 //! to store stakes.
 
-pub mod adapter;
 mod stake_store;
-mod store_pruner;
 
 pub use stake_store::StakeStorer;
```

Comment:
We keep StakeStore that contain the trait. 
Maybe it's could be better to move this trait from `persistence` to `aggregator` and `signer`
What do you think ?